.github/ISSUE_TEMPLATE/NEW_RELEASE.md

```diff
@@ -41,10 +41,15 @@ Please do not remove items from the checklist
       to production: <!-- example kubernetes/k8s.io#3612-->
 - [ ] Wait for the PR to be merged and verify that the image `registry.k8s.io/kueue/kueue:$VERSION` is available.
 - [ ] Publish the draft release prepared at the [Github releases page](https://github.com/kubernetes-sigs/kueue/releases).
-      Link: <!-- example https://github.com/kubernetes-sigs/kueue/releases/tag/v0.1.0 -->
+      <!--  Link:  example https://github.com/kubernetes-sigs/kueue/releases/tag/v0.1.0 -->
 - [ ] For major or minor releases, merge the `main` branch into the `website` branch to publish the updated documentation.
-- [ ] Send an announcement email to `sig-scheduling@kubernetes.io` and `wg-batch@kubernetes.io` with the subject `[ANNOUNCE] kueue $VERSION is released`. Link: <!-- example https://groups.google.com/a/kubernetes.io/g/wg-batch/c/-gZOrSnwDV4 -->
-- [ ] Update `README.md`, `CHANGELOG`, `site/config.toml`, `charts/kueue/Chart.yaml` (`appVersion`) and `charts/kueue/values.yaml` (`controllerManager.manager.image.tag`) in `main` branch: <!-- example #774 -->
+- [ ] Send an announcement email to `sig-scheduling@kubernetes.io` and `wg-batch@kubernetes.io` with the subject `[ANNOUNCE] kueue $VERSION is released`.   <!--Link: example https://groups.google.com/a/kubernetes.io/g/wg-batch/c/-gZOrSnwDV4 -->
+- [ ] Update the below files with respective values in `main` branch : 
+- Latest version in `README.md`
+- Release notes in the `CHANGELOG`
+- `version` in `site/config.toml`
+- `appVersion` in `charts/kueue/Chart.yaml` 
+- `last-updated`, `last-reviewed`, `commit-hash`, `project-release` in `SECURITY-INSIGHTS.yaml`
```

Comment:
Should we add `distribution-points` as well?

Reply:
Can you specify in which file we have to add `distribution-points` ? I'm not able to find that particular value

Reply:
I meant `SECURITY-INSIGHTS.yaml`.

Reply:
Ohh got it! I think we also have to change there. Updating it.